Sources/RxViewController/UIViewController+Rx.swift

```diff
@@ -50,5 +50,39 @@ public extension Reactive where Base: UIViewController {
     let source = self.methodInvoked(#selector(Base.didReceiveMemoryWarning)).map { _ in }
     return ControlEvent(events: source)
   }
+
+    /// Rx observable, triggered when the ViewController has appeared for the first time
+    public var firstTimeViewDidAppear: Single<Void> {
+        return self.sentMessage(#selector(Base.viewDidAppear)).map{_ in return Void()}.take(1).asSingle()
+    }
+
+    /// Rx observable, triggered when the ViewController appearance state changes (true if the View is being displayed, false otherwise)
+    public var isBeingDisplayed: Observable<Bool> {
+        let viewDidAppearObservable = self.base.rx.viewDidAppear.map { _ in true }
+        let viewWillDisappearObservable = self.base.rx.viewWillDisappear.map { _ in false }
+        return Observable<Bool>.merge(viewDidAppearObservable, viewWillDisappearObservable)
+    }
+
+    /// Rx observable, triggered when the ViewController is being dismissed
+    public var isDismissed: ControlEvent<Bool> {
+        let source = self.sentMessage(#selector(Base.dismiss)).map { $0.first as? Bool ?? false }
+        return ControlEvent(events: source)
+    }
+
+    /// Rx observable, triggered when the ViewController or its parent view is being dismissed
+    public var selfOrParentIsDismissed: Observable<Void> {
```

Comment:
Can you share a use case of this property? I have no idea about the purpose of it.